docs/Getting_Started.md

```diff
@@ -0,0 +1,421 @@
+# Getting started with Discord app development
+
+Discord apps let you customize your servers with interactions and automation. This guide is meant to walk through building and running your first Discord app using JavaScript. At the end of this guide, you’ll have an app with a bot user that uses slash commands, sends messages, and interacts with message components.
+
+While this guide is beginner-focused, it assumes a basic understanding of [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics).
+
+> info
+> When developing apps, you should build and test in a server that isn’t actively used by others. If you don’t have your own server already, you can [create one for free](https://discord.com/developers/teams).
```

Comment:
why does it take to `teams` page? isn't it saying about create a new server?

Reply:
ahhhh I'm not sure how that got in there, thank you 👀